### ISSUE TITLE
Show ETH/BTC on comparison chart

### ISSUE THREAD

**Issue by czxtm** [icostats/icostats#19]

**Comment by felix2feng** [comment]
@coopermaruyama Does this mean adding ETH and BTC lines (pegged to the earliest ICO starting date) to the "Compare" tab?

**Comment by coopermaruyama** [comment]
@felix2feng  yep. Currently the logic is written so the 'beginning' of the time range is determined by which ICO's are 'selected' (whichever selected ICO had the earliest ICO date).  That means this will require some extra logic in the function that generates these data points so that it stays that way even when ETH/BTC is selected.

Also, ETH/BTC should probably be in the default items. Right now they are just some arbitrarily picked ICOs.
